sass/a11y-en.scss

```diff
@@ -1,7 +1,8 @@
 @charset 'UTF-8';
 
+$messages: require('./locales/en', $infer: false);
```

Comment:
Instead of setting a locale that tells Sass where to look in the `$messages` map, we dynamically define the `$messages` map with translations for the current language exclusively thanks to Sassport.